Add maxlength minlength examples

vipulgupta2048 opened the pull request:
:hatching_chick: 
Added missing examples for `maxlength` and `minlength` validation rules to the docs in accordance with #500, let me know if there are any more examples that you like to see. Or improve any examples. Would be happy to write and contribute to the docs. 

Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

funkyfuture commented on docs/validation-rules.rst:
```diff
@@ -457,6 +458,22 @@ minlength, maxlength
 
 Minimum and maximum length allowed for iterables.
```
on that occassion you could correct one of my mistakes: "iterables" is false in that sentence. truth is that any object that implements `__len__` works with this rule. the abstract class is `collections.abc.Sized`, but i don't think the term "sized" is established and clear.

vipulgupta2048 replied:
True that in my project to we had a bit of trouble with collections as we were trying to validate Cerberus Schema and on first thought I just went to validate if the schema is a dict or not. But, later after digging into Mapping and Cerberus's source code that I realized that any Mapping type can be validated.  

I have changed it now let me know if this works.

funkyfuture commented:
also, could you please change your branch to base on the [`1.3.x`](https://github.com/pyeve/cerberus/tree/1.3.x) branch and update the pr accordingly.

vipulgupta2048 commented on AUTHORS:
```diff
@@ -59,6 +59,7 @@ Contributors
 - Waldir Pimenta
```
Implemented, thanks.

vipulgupta2048 commented on docs/validation-rules.rst:
```diff
@@ -444,7 +444,8 @@ The assigned data can be of any type.
 --------
 
-Minimum and maximum value allowed for any types that implement comparison operators.
+Minimum and maximum value allowed for any types that implement comparison operators. See
```
The proposal seems fine, but not sure what are ` (__lt__ or __gt__)`, I can just add them if you want. I just felt that by comparison operators we are going for `\*of-rules`

Roger that, a simple example coming right up.

funkyfuture replied:
these "dunder" (from double underscore) or also known as "magic" methods do implement these *operations* that the *operators* imply. so, maybe just "comparison operations" instead of "operator"?

vipulgupta2048 replied:
Thanks, @funkyfuture for the explanation.

vipulgupta2048 commented on docs/validation-rules.rst:
```diff
@@ -457,6 +458,22 @@ minlength, maxlength
+    False
+
+    >>> v.errors
+    >>> {'numbers': ['max length is 3']}
```
Yes, must have missed that. Done.

vipulgupta2048 commented:
Hey @funkyfuture, I guess it's a first time for everything. I have changed the base branch (First time) and implemented the changes. Let me know how you feel about them.
Thanks, ,,

funkyfuture commented:
thanks, merged with 0fccd9bb9186373582565ef0cfa73844e7f33173
